static/app/components/forms/types.tsx

```diff
@@ -48,6 +48,7 @@ interface BaseField {
   disabled?: boolean | ((props: any) => boolean);
   disabledReason?: React.ReactNode | ((props: any) => React.ReactNode);
   extraHelp?: string;
+  feature?: string;
```

Comment:
i don't think adding this here makes sense since it doesn't apply to all the places BaseField is used